e2e/wdio/core/tests/step-input.e2e-spec.ts

```diff
@@ -0,0 +1,159 @@
+import { StepInputPo } from '../pages/step-input.po';
+import {
+    refreshPage,
+    getElementArrayLength,
+    click,
+    getValue,
+    doubleClick,
+    pause,
+    getText,
+    clearValue,
+    setValue,
+    getElementClass,
+    sendKeys,
+    isElementDisplayed
+} from '../../driver/wdio';
+import { sections } from '../fixtures/appData/step-input-content'
+
+describe('Time component test', function () {
+    const stepInputPage = new StepInputPo();
+    const {
+        formExample,
+        labelExample,
+        localExample,
+        stateExample,
+        configExample,
+        defaultExample,
+        currencyExample,
+        step,
+        input,
+        plusButton,
+        minusButton,
+        text,
+        textForDisabledExample
+    } = stepInputPage;
+
+    beforeAll(() => {
+        stepInputPage.open();
+    }, 1);
+
+    afterEach(() => {
+        refreshPage();
+    }, 2);
+
+    it('should check default example inputs', () => {
+        for (let i = 0; i < sections.length; i++) {
+            checkInputWorking(sections[i], 'click', 'positive')
+            checkInputWorking(sections[i], 'click', 'negative')
+            checkInputWorking(sections[i], 'type', 'negative')
+            checkInputWorking(sections[i], 'type', 'positive')
+        }
+    })
+
+    it('should check limitation for step-input', () => {
```

Comment:
unclear. you say limitation, but you are expecting class to contain disabled, i don't get it

Reply:
in this case maximum value is 10, so if value is 10 - plus button should be disabled. The same with minimum (-10). Anyway i did it more clearly

Reply:
aha, ok, so please rename to something like 'should check +/- button disabled when input limit reached'

also need to add another test case
it('should not allow input above/below max') and add value by sending keys 20, -20 hit enter key and check input field shows 10, -10